src/psp_lda.py

```diff
@@ -0,0 +1,112 @@
+import sys
```

Comment:
For organizing the entire repo, it would be great to put all of the LDA related files into one subdirectory of src/, perhaps src/lda. We also might want to rename these files so they better describe what they do, versus naming them by who wrote them.